src/components/MoneyReportHeader.js

```diff
@@ -92,23 +108,43 @@ function MoneyReportHeader(props) {
                         />
                     </View>
                 )}
+                {shouldShowApproveButton && !isSmallScreenWidth && (
+                    <View style={[styles.pv2]}>
+                        <Button
+                            success
+                            text={translate('iou.approve')}
+                            style={[styles.mnw120]}
+                            onPress={() => IOU.approveMoneyRequest(moneyRequestReport)}
+                        />
+                    </View>
+                )}
             </HeaderWithBackButton>
-            {shouldShowSettlementButton && props.isSmallScreenWidth && (
-                <View style={[styles.ph5, styles.pb2, props.isSmallScreenWidth && styles.borderBottom]}>
+            {shouldShowSettlementButton && isSmallScreenWidth && (
```

Comment:
Will we ever show Settlement and Approve button at the same time?

Reply:
No I think the conditionals implemented will prevent this, this specifically `const isPayer = policyType === CONST.POLICY.TYPE.CORPORATE ? isPolicyAdmin && isApproved`